### ISSUE TITLE
mAP of EfficientDet- d1 model is so low.

### ISSUE THREAD

**Issue by quangtn266** [toandaominh1997/EfficientDet.Pytorch#107]
I tried to train efficientdet -d1 model on VOC dataset and got low mAP.
mAP:
aeroplane: 0.37774546979552437
bicycle: 0.2717609619189093
bird: 0.07373408655526852
boat: 0.11174968193861216
bottle: 0.03085867327429357
bus: 0.3259476835112142
car: 0.4270658513682306
cat: 0.1865584138729217
chair: 0.039742404670251544
cow: 0.08904280802274998
diningtable: 0.1571207438493799
dog: 0.11370612854293503
horse: 0.3255647249082447
motorbike: 0.32683360884097407
person: 0.2828047853052909
pottedplant: 0.006887167196655821
sheep: 0.13986957637436875
sofa: 0.15581529346760506
train: 0.36374159709309817
tvmonitor: 0.2485077007219078
avg mAP: 0.20275286806142181

I waited for the 174th epoch, but mAP wasn't improved, so I didn't think that this issue is related to training time.
I also tried to change hyper- parameters such as learning rate, batch size,... and augmentation such as common size = 640 and scale parameter in augmentation.py, but mAP didn't be improved. Has anyone any solution for this problem?
